google-login-plugin/build.gradle

```diff
@@ -85,3 +86,8 @@ pmd {
     sourceSets = [sourceSets.main]
 }
 
+processResources {
+    filter ReplaceTokens, tokens: [
+            "usage.tracker.property": System.getProperty("usageTrackerProperty")
```

Comment:
analytics.id?

also environment variables should be constant case and project namespaced. Something like GCLOUD_INTELLIJ_ANALYTICS_ID?